docs/source/usersguide/input.rst

```diff
@@ -528,6 +528,8 @@ attributes/sub-elements:
     sub-elements/attributes are those of a univariate probability distribution
     (see the description in :ref:`univariate`).
 
+    *Default*: Watt spectrum with :math:`a` = 0.988 MeV and :math:`b` = 2.249 MeV
```

Comment:
Whoops, so it is.  Relatedly, is it possible to make one of these `*Default*` statements with a line-break in it?  When I try with a line-break, it makes the statement bold.